examples/v1alpha1/pipelineruns/conditional-pipelinerun-continue-after-skip-resource.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Condition
+metadata:
+  name: is-equal
+spec:
+  params:
+  - name: left
+    type: string
+  - name: right
+    type: string
+  check:
+    image: alpine
+    script: |
+      #!/bin/sh
+      if [ $(params.left) = $(params.right) ]; then
+        echo "$(params.left) == $(params.right)"
+        exit 0
+      else
+        echo "$(params.left) != $(params.right)"
+        exit 1
+      fi
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: sum
+  annotations:
+    description: |
+      A simple task that sums the two provided integers
+spec:
+  inputs:
+    params:
+      - name: a
+        type: string
+        default: "1"
+        description: The first integer
+      - name: b
+        type: string
+        default: "1"
+        description: The second integer
+  results:
+    - name: sum
+      description: The sum of the two provided integers
+  steps:
+    - name: sum
+      image: bash:latest
+      script: |
+        #!/usr/bin/env bash
+        echo -n $(( "$(inputs.params.a)" + "$(inputs.params.b)" )) | tee $(results.sum.path)
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: multiply
+  annotations:
+    description: |
+      A simple task that multiplies the two provided integers
+spec:
+  inputs:
+    params:
+      - name: a
+        type: string
+        default: "1"
+        description: The first integer
+      - name: b
+        type: string
+        default: "1"
+        description: The second integer
+  results:
+    - name: product
+      description: The product of the two provided integers
+  steps:
+    - name: product
+      image: bash:latest
+      script: |
+        #!/usr/bin/env bash
+        echo -n $(( "$(inputs.params.a)" * "$(inputs.params.b)" )) | tee $(results.product.path)
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: condition-pipeline
+spec:
+  params:
+  - name: a
+    type: string
+  - name: b
+    type: string
+  tasks:
+  - name: sum-inputs # condition evaluates to false, should be skipped (failed)
+    conditions:
+    - conditionRef: is-equal
+      params:
+        - name: left
+          value: "1"
+        - name: right
+          value: $(params.b)
+    continueAfterSkip: 'true'
+    taskRef:
+      name: sum
+    params:
+      - name: a
+        value: "$(params.a)"
+      - name: b
+        value: "$(params.b)"
+  - name: multiply-inputs # should execute (succeeded)
+    taskRef:
+      name: multiply
+    params:
+      - name: a
+        value: "$(params.a)"
+      - name: b
+        value: "$(params.b)"
+  - name: sum-and-multiply # should execute, resolution error (failed)
```

Comment:
usually we don't have examples that fail, cuz we want the examples to show successful runs that folks can copy